force-app/main/default/layouts/Contact-Contact %28Marketing%29 Layout.layout-meta.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I am not sure if adding or modifying the standard layouts is a good idea! If someone installs apex-recipes in production as an unlocked package (or deploying it), there will be overrides.

Reply:
I agree with @msrivastav13, I don't recommend it. If I recall well, there was some of that in the original Dreamhouse and it ended "badly".

Reply:
yeah, I'm removing the layout it was an oversight and shouldn't have been added.